### DISCUSSION TITLE
Seasonal performance leaderboard?

### DISCUSSION THREAD

**Discussion by CuredDDN** [ppy/osu#27841] [Ideas]
I was watching the new osu lazer video release and saw seasons within the video. Now basically, this is basically the osu performance board, but instead the seasonal leaderboards only contains performance points and beatmaps that were earned during the season. And when the season ends, the entire seasonal performance leaderboard gets reset Now I am unsure if this is a hard task or not (me personally I think this would be difficult) to do, but I think this would inspire players to play more competitively as the number one spot could change more frequently and gives player a chance for a number one spot, although seasonally.

**Comment by cihe13375** [comment]
Some extra thoughts from me:
1) maybe the seasonal performance leaderboard should prioritize maps ranked recently (e.g. if the current season is October-December, then maybe the maps ranked in July-September get prioritized; by "prioritize" I mean, for example, at least 25% of "seasonal pp" must come from them). This is more in line with the idea of "season" and can encourage players to play new maps (I recall that when I started playing osu! every map had like 100k+ pc, but now I have seen many ranked maps not even having 10k pc). Btw, without this design the players would only need to farm the same maps at the beginning of every season, which is not the right direction imo.
2) the player can *optionally* select seasonal ranking to be displayed in the user info card / userpage; this is to provide motivation to compete on the seasonal leaderboard. The users can still choose to display the original, fully accumulative ranking if they don't want to participate in the continuous seasonal war.

**Reply by CuredDDN** [reply]
I really like those ideas, also one more.

When the season ends, they could save the leaderboard maybe from top 1 to 1000 or 100. And make it into a spotlight like what is already in osu.

Also I was on reddit earlier, and someone was mentioning clans which I also thought was a cool idea
